local-volume/provisioner/pkg/util/volume_util.go

```diff
@@ -99,6 +104,18 @@ func (u *volumeUtil) DeleteContents(fullPath string) error {
 	return nil
 }
 
+// GetFsAvailableByte returns available capacity in byte about a mounted filesystem.
+// fullPath is the pathname of any file within the mounted filesystem. Capacity
+// returned here is total capacity available to non-root users, and does not include
+// fs reserved space.
+func (u *volumeUtil) GetFsAvailableByte(fullPath string) (uint64, error) {
```

Comment:
Thanks for pointing this out.  I didn't know there is such a util in kubernetes :)  You can take a look at discussion here: https://github.com/kubernetes-incubator/external-storage/pull/135#discussion_r119220717   Basically, available from that package returns all available spaces block wise, but there are certain amount of blocks can't be used by normal user. AvailableByte removes those.